### ISSUE TITLE
[Development] `logsmith-monitor` Support

### ISSUE THREAD

**Issue by TanmoySG** [TanmoySG/logsmith#4]
### logsmith-monitor support for logsmith

- From [@TanmoySG/logsmith-monitor](https://github.com/TanmoySG/logsmith-monitor)
- Tracking Issues
  - [x] https://github.com/TanmoySG/logsmith-monitor/issues/38
  - [x] https://github.com/TanmoySG/logsmith-monitor/issues/37
- Checkpoints
  - [x] #5 
  - [x] #6 
  - [x] #7 
  - [x] #8 
  - [x] #11

**Comment by TanmoySG** [comment]
https://github.com/TanmoySG/logsmith-monitor/issues/38#issuecomment-1126673402

Save logs to File only on Exit. R/W from program storage/variable.

**Comment by TanmoySG** [comment]
Using branch [`dev`](https://github.com/TanmoySG/logsmith/tree/dev)

**Comment by TanmoySG** [comment]
## Possible Feature/Task List

Will be formulated into proper issues.

Primarily the tasks would be broadly to Port logsmithJs methods/code to python

- [x] #5 
- [x] #6 
- [x] #7 
- [x] #8 
- [ ] #11

**Comment by TanmoySG** [comment]
String Templating 

```
sample = {
    "name" : "tanmoy",
    "age" : "22",
    "gen" : "F"
}

template_String = "My Name is {name} gender {gen}".format(**sample)

print(template_String)
```

https://stackoverflow.com/questions/45806285/python-how-to-format-a-string-based-on-dict-or-json-object

**Comment by TanmoySG** [comment]
```
sample = {
    "name" : "tanmoy",
    "age" : "22",
    "gen" : "F"
}

template_String = "My Name is {name} gender {gen}".format(**sample)

def template_filler(x):
    return "My Name is {name} gender {gen}".format(**x)

temp_helper = template_filler

print(temp_helper(sample))
```

**Comment by TanmoySG** [comment]
### Published `v0.1.0` with monitor support!
- [PyPi] https://pypi.org/project/logsmith/
- [Docs] https://github.com/TanmoySG/logsmith#readme
- [Release] https://github.com/TanmoySG/logsmith/releases/tag/v0.1.0
- [Tag] https://github.com/TanmoySG/logsmith/tree/v0.1.0

**Comment by TanmoySG** [comment]
Closes https://github.com/TanmoySG/logsmith-monitor/issues/38
